src/screen.rs

```diff
@@ -42,6 +44,9 @@ pub struct PrettyPrompt {
 impl Interfaceable for PrettyPrompt {
     fn new() -> PrettyPrompt {
         let mut rl = Editor::<()>::new();
+        if !Path::new(".history.txt").exists() {
```

Comment:
Low-key, is there a way to do history without a file?

Reply:
Twould be nice, I'll see if I can figure that out.